<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Objeto</class>
 <widget class="QWidget" name="Objeto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
   <item>
    <widget class="QLabel" name="lblImage">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="objetoLabel">
       <property name="text">
        <string>Objeto</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="led_objeto"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="legendaLabel">
       <property name="text">
        <string>Legenda</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="led_legenda"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="proprietRioLabel">
       <property name="text">
        <string>Propriet√°rio</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="led_proprietario"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
